src/network/networkd-link.c

```diff
@@ -3052,6 +3091,15 @@ int link_update(Link *link, sd_netlink_message *m) {
                                 return r;
                         }
                 }
+
+                if (link->radv) {
+                        r = sd_radv_set_mtu(link->radv, link->mtu);
+
+                        if (r < 0) {
+                                log_link_warning_errno(link, r, "Could not set MTU for Router Advertisment: %m");
+                                return r;
```

Comment:
you can merge these two lines in one: `return log_link_warning_errno(link, r, …)`